src/components/navbar.tsx

```
@@ -51,11 +51,9 @@ export function NavBar() {
   // Check if user is admin
   useEffect(() => {
     async function checkAdminStatus() {
-      if (user && !isDemoMode) {
-        // HARDCODED: Double-check both role and email
-        const role = await getUserRole(user.id);
-        const isAuthorizedAdmin = isHardcodedAdmin(user.email || '');
-        setIsAdmin(role === 'admin' && isAuthorizedAdmin);
+      if (user && user.email && !isDemoMode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)